packages/cxl-ui/scss/cxl-app-layout/_layout.scss

```diff
@@ -23,6 +23,11 @@ $toggle-icon: "lumo:angle-right";
 :host([layout^="2c"]) {
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      */
+      min-height: 90%;
```

Comment:
I used 90% here as 100 caused a scrollbar to appear.

Reply:
1. that's code-level comment material.
2. scrollbar === it's an incorrect solution, or some other layout element needs adjustment, too.